geocoordinates/geocoordinates.php

```diff
@@ -26,34 +26,40 @@ function geocoordinates_uninstall()
 
 function geocoordinates_resolve_item(&$item)
 {
-	if((!$item["coord"]) || ($item["location"]))
+	if((!x($item, 'coord')) || (x($item, 'location')))
```

Comment:
Apparently you didn't get my comment, so here you are: https://friendica.mrpetovan.com/display/735a2029915af44c77d5aee120922286

Reply:
Your comment hadn't arrived at the forum. Could you have a look at your logfile, why?

Reply:
I sent you a private message about this issue, did you get it? Am I speaking into the void on my node?

Reply:
It seems so. I hadn't got any message from you. Please have a look at your logfile (everything with DFRN.php) while sending. And we shouldn't discuss this here in this PR.

Reply:
I'll look in about 3 hours.